.happy/terraform/modules/sfn_config/gisaid.wdl

```diff
@@ -109,16 +113,14 @@ task IngestGISAID {
     aspen_workflow_rev=$COMMIT_SHA
     aspen_creation_rev=$COMMIT_SHA
 
-    # fetch genepi config
-    sequences_key="raw_gisaid_dump/${build_id}/gisaid.ndjson.zst"
+    # Set our destination key
+    sequences_key="raw_genbank_dump/${build_id}/genbank.ndjson.zst"
 
-    # fetch the gisaid dataset and transform it.
-    wget "~{gisaid_ndjson_url}" --user "${gisaid_username}" --password "${gisaid_password}" --continue --tries=2 -O gisaid_dump.fasta.bz2
-    bunzip2 gisaid_dump.fasta.bz2 
-    zstdmt gisaid_dump.fasta -o sequences.fasta.zst
-    rm gisaid_dump.fasta
-
-    ${aws} s3 cp sequences.fasta.zst "s3://${aspen_s3_db_bucket}/${sequences_key}"
+    # fetch the genbank dataset and transform it.
+    # ${aws} s3 cp "~{upstream_ndjson_url}" genbank.ndjson.zst
```

Comment:
I'd like to switch to downloading these files via S3 instead of HTTPS -- nexstrain makes them available via a public s3 bucket, and it's cheaper/better for everyone to fetch them directly from the bucket. I was having problems with the batch job's IAM configuration so I fell back to HTTPS for now. I'm leaving the comments in as a reminder to debug this further in the future